.azure/pipelines/ci.yml

```diff
@@ -10,36 +10,13 @@ trigger:
     - master
     - release/*
     - internal/release/*
-  paths:
-    exclude:
-    - .github/*
-    - .vscode/*
-    # Exclude only a few specific Markdown files because some of them ship.
-    - docs/*
-    - CODE-OF-CONDUCT.md
-    - CONTRIBUTING.md
-    - LICENSE.TXT
-    - README.md
-    - SECURITY.md
-    # Do not ignore the THIRD-PARTY-NOTICES.TXT file because it ships.
 
 # Run PR validation on all branches
 pr:
   autoCancel: true
   branches:
     include:
     - '*'
-  paths:
-    exclude:
-    - .github/*
-    - .vscode/*
-    - '**/*.md'
-    - CODE-OF-CONDUCT.md
-    - CONTRIBUTING.md
-    - LICENSE.TXT
-    - README.md
-    - SECURITY.md
-    - THIRD-PARTY-NOTICES.TXT
```

Comment:
These exclusions were **Just Wrong:tm:** because our branch protection rules require a successful run of this pipeline